### ISSUE TITLE
[Suggestions] 

### ISSUE THREAD

**Issue by wseery** [AnthonyOSX/GO-Bot#7]
Allow people to see the screen (maybe a lower quality picture or something)
Make more things customizable:
-which items to delete
-which pokemon to delete
-power range for pokemon to delete
-which pokemon to catch

**Comment by MarcusOy** [comment]
Technically, the bot is interfacing with the API of the game. It is not emulating the Pokemon Go app itself, so a picture cannot be rendered. If the Dev would want to have something that is sorta like the app, he would have to create a Google Maps applet with coordinates showing where the bot currently is farming.

I'm not very familiar with the API, but is there a function to see your inventory? I could possibly create an interface for that if that function existed.

**Comment by wseery** [comment]
@marcusoy ok, so maybe you could get the stats for the account and see what pokemon/eggs it has and what items?

**Comment by AnthonyOSX** [comment]
@MarcusOy @wseery You can open Pokemon GO on your phone while the bot is running.

**Comment by MarcusOy** [comment]
@AnthonyOSX Yeah, been running the bot for less than an hour and im already level 6. But it would still be nice to fill in the extra space with some account stats and inventory.

**Comment by AnthonyOSX** [comment]
@MarcusOy Planning on using that space for bot options (transfer Pokemon, use berries, etc.) shortly :)

**Comment by wseery** [comment]
@AnthonyOSX opening it while using the bot won't get you banned?  Like they won't see that you are in 2 places at once?

**Comment by wseery** [comment]
@AnthonyOSX  i haven't used the bot enough yet to see this, but what happens when it fills up with pokemon or items and has no more space?

**Comment by MarcusOy** [comment]
@wseery My guess is it stops collecting stuff. If you do hit the cap, just delete some items in the app.

**Comment by wseery** [comment]
@MarcusOy any idea if you can log out of your main account and log into your alt on your phone without getting banned?  I don't have access to another device to use to check

**Comment by spot88** [comment]
http://en.bignox.com/pokemongo - Use this

**Comment by MarcusOy** [comment]
@wseery I created a dedicated bot account and I am allowed to freely log in to my main and the bot account. I can confirm that I haven't been banned, but Niantic might actually be smart about it and ban bot users and their phones at a later date.

**Comment by Minnieal28** [comment]
Could the ability to log in with your Google account be added?

**Comment by dgol1115** [comment]
One suggestion I have is to add the ability to turn off auto transfer. It's not the cleanest method, I've noticed plenty of high level pokemon get transferred out, and Id rather handle it on my own, of possible.

**Comment by sigboe** [comment]
@AnthonyOSX Hi there, I made just a quick mockup of how more information could possibly be displayed your program in the future.

![image](https://cloud.githubusercontent.com/assets/2087930/17058296/fd59abaa-501f-11e6-9b2c-692eabdc1560.png)

**Comment by MarcusOy** [comment]
@sigboe Cool mockup. Assuming the ATI he is using can also display account stats and inventory, this should work. About the live map, we would have to combine another program called Pokémon Live Map, which is a python based script that reveals Pokémon at a location. Although it is possible, it would take a while to pull of and he would need the permission of the creator of the live map.

**Comment by DuIslingr** [comment]
@MarcusOy Considering the bot already keeps a set number of pokeballs and such in order to detemrine when to recycle items, I would say that getting inventory amount and being able to recycle items through the bot should be possible. As for stats, I am unsure at this point its entirely possible, but seeing as the bot does currently know what level you are and how much xp to go, that much at least could be done like the mock up shows.

**Comment by Majamulli** [comment]
I'll write this here as well since suggestions should all be kept here. I would like a list of the pokemon I currently have with the IV and CP so I can chose which to evolve myself. As it is right now it's showing the pokemon with the highest CP and highest IV so it shouldn't be too hard to just show them all.
And I am against automatically renaming them with the IV because I don't want all my pokemon to have weird names. If that gets implemented, make it so that I can choose whether I want it activated or not please.
